src/PKSim.UI.Starter/PKSimStarterUserInterfaceRegister.cs

```diff
@@ -12,10 +13,37 @@ public override void RegisterInContainer(IContainer container)
          container.AddScanner(scan =>
          {
             scan.AssemblyContainingType<UserInterfaceRegister>();
-            scan.WithConvention<PKSimStarterRegistrationConvention>();
+            scan.WithConvention<PKSimRegistrationConvention>();
+            scan.Include(shouldRegisterType);
          });
          container.Register<OSPSuite.UI.Services.IToolTipCreator, IToolTipCreator, ToolTipCreator>(LifeStyle.Transient);
          container.Register<IExitCommand, ExitCommand>();
       }
+
+      private bool shouldRegisterType(Type concreteType)
```

Comment:
no not like this. I don't understand why we would incluise the namespaces by name and not using type
=>IncudeNamespacingContaining<X>

Reply:
scan.IncludeNamespaceContainingType<>();

Reply:
Oh, I see what you're getting at. concreteType.FullName will include the namespace. So anything in ExpressionProfile namespace will be registered by this. But also, anything where the class includes 'ExpressionProfile' in the class name that's not in the 'ExpressionProfile' namespace. For example, if there's a mapper.

I can revise for dedicated namespace scanning.

Reply:
Yeah, this is registering a mixture of components from various namespaces some with class names that match, some which match the namespace.